tensorflow/python/ops/math_ops.py

```diff
@@ -1144,6 +1144,14 @@ def multiply_no_nan(x, y, name=None):
 
   Returns:
     The element-wise value of the x times y.
+
+  Usage Example:
```

Comment:
"Usage Example" is to be added before "Args" and "Returns" blocks as is done in the entire file. Please shift the code accordingly.

Reply:
@Jake-Short Please do the same for all the usage examples you've added.